src/app/[locale]/specs/oauth/en.mdx

```diff
@@ -189,6 +189,10 @@ The intention is to support flexible scopes based on Lexicon namespaces (NSIDs)
 - `transition:email`: access to the account email address
     - email address (and confirmation status) gets included in response to `com.atproto.server.getSession` endpoint
 
+<Note>
+As of August 2025, a more granular permission system is available for use, announced and described [here](https://github.com/bluesky-social/atproto/discussions/4118). This mechanism has not yet been formally specified, but the design is considered stable.
+</Note>
+
```

Comment:
I guess this needs translating for the Korean version too?

Reply:
> This mechanism has not yet been formally specified
I think this phrasing makes things sound a little more open-ended/incomplete than they are at the moment.  Maybe we can tweak it a bit?